draft-beck-tls-trust-anchor-ids.md

```diff
@@ -168,7 +195,7 @@ Subscribers are configured with one or more candidate certification paths to pre
 
 Each candidate path which participates in this protocol must be provisioned with the trust anchor identifier for its corresponding trust anchor.
 
-To carry this information, this document defines the `trust_anchor_identifier` property for a CertificatePropertyList {{Section 5 of !I-D.davidben-tls-trust-expr}}. This `data` field of the property contains the binary representation of the trust anchor identifier of the certification path’s trust anchor, as described in {{trust-anchor-ids}}.
+To carry this information, this document defines the `trust_anchor_identifier` property for a CertificatePropertyList. This `data` field of the property contains the binary representation of the trust anchor identifier of the certification path’s trust anchor, as described in {{trust-anchor-ids}}.
 
 ~~~
 enum {
```

Comment:
We end up defining this enum twice, one time with value zero and another time with value TBD. :-) Perhaps we just delete this copy?

Reply:
yeah removed this one and hoisted some of the text up.